src/components/App/App.js

```diff
@@ -50,6 +50,14 @@ class App extends Component {
                 </PublicRoute>
               }
             />
+            <Route
+              path="/login/shelter"
+              element={
+                <PublicRouteShelter>
+                  <ShelterLoginPage />
```

Comment:
I am guessing that you can do something like <LoginPage userType="shelter"/>. If userType is null/undefined, set the page for a user, if userType is shelter, set the page for shelter.